server/sql_migrations/20200602163951_create_maintenance_requests.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS public.nsl_maintenance;
```

Comment:
Can we actually rename this `maintenance_requests`?

Reply:
renamed table to maintenance_requests